project/api/models.py

```diff
@@ -1,27 +1,52 @@
 from django.core.exceptions import ValidationError
 from django.contrib.gis.db import models
 from simple_history.models import HistoricalRecords
+from colorfield.fields import ColorField
 
 # Create your models here.
 class Nation(models.Model):
-    name = models.SlugField(max_length=20)
-    local_name = models.CharField(max_length=30)
-    wikipedia = models.URLField()
+    """
+    Cultural/governmental entity. Serves as foreign key for most Territories
+    """
+    name = models.CharField(max_length=20,
+                            help_text="Canonical name, should not include any epithets, must be unique",
+                            unique=True)
+    color = ColorField(default="#FF0000",
+                       help_text="Color to display on map",
+                       unique=True)
     history = HistoricalRecords()
 
+    #Flavor fields
+    aliases = models.TextField(help_text="CSV of alternative names this state may be known by",
```

Comment:
My thinking when I made this field in the previous version of this backend was the example of what we call the "Byzantine Empire" or "Eastern Roman Empire" or "Byzantium", which mainly thought of itself as the "Roman Empire" or "Romania". In this case we would pick one of the names as the canonical name (probably "Byzantine Empire" to match Wikipedia) and list the others in this aliases CSV.

Reply:
^, thoughts?

Reply:
Got it, however I am not 100% convinced if its our project's scope to put the aliases, or wikipedia's scope. Though it can't do any bad to have a little bit of extra data in our db, so let's keep it.

What does bother me is that it's a TextField. Postgres allows Text[] columns, it'd fit much better than a serialized csv string (see [reasons](https://stackoverflow.com/questions/3653462/is-storing-a-delimited-list-in-a-database-column-really-that-bad))

Reply:
That makes sense. I'd support a conversion to an ArrayField like this: https://docs.djangoproject.com/en/2.1/ref/contrib/postgres/fields/#arrayfield

Reply:
Nice catch, I'll convert the field